Content.Shared/Roles/StartingGearPrototype.cs

```diff
@@ -45,6 +55,17 @@ public interface IEquipmentLoadout
     /// </summary>
     public Dictionary<string, EntProtoId> Equipment { get; set; }
 
+    /// <summary>
+    /// Overides the players entity
+    /// </summary>
+    public string? Entity { get; set; }
+
+    /// <summary>
+    /// Meant to be used in conjunction with Entity
+    /// Entity to show as the players dummy in the lobby
+    /// </summary>
+    public string? EntityDummy { get; set; }
+
     /// <summary>
```

Comment:
This should be on loadoutprototype, not startinggear.
StartingGear should just be about applying equipment to an entity not replacing it entirely.